app/code/Magento/AsynchronousOperations/Api/GetAllOperationsInterface.php

```diff
@@ -0,0 +1,24 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\AsynchronousOperations\Api;
+
+use Magento\AsynchronousOperations\Api\Data\OperationInterface;
+
+/**
+ * Interface for load all operations from a bulk
+ */
+interface GetAllOperationsInterface
```

Comment:
There is no need to introduce new interface.

Reply:
removed